integrations/mastra/typescript/src/mastra.ts

```diff
@@ -184,6 +185,7 @@ export class MastraAgent extends AbstractAgent {
                   const memory = await this.agent.getMemory();
                   if (memory) {
                     const workingMemory = await memory.getWorkingMemory({
+                      resourceId: this.resourceId,
```

Comment:
Everything is 'resource' scoped now, so we need to pass it here: https://mastra.ai/guides/v1/migrations/upgrade-to-v1/memory#default-memory-scope-from-thread-to-resource

That's why I made it required everywhere